pyvera/__init__.py

```diff
@@ -637,7 +637,10 @@ def get_color_index(self, colors, refresh=False):
         ci = None
         sup = self.get_complex_value('SupportedColors')
         if sup is not None:
-            ci = [sup.split(',').index(c) for c in colors]
+            try:
```

Comment:
The list comprehension was making my head hurt, but the fix is quite simple I think:
`ci = [sup.split(',').index(c) for c in colors if c in a]`

tested in the console below:
```
>>> sup
'W,A,I'
>>> colors
['R', 'G', 'B']
>>> ci = [sup.split(',').index(c) for c in colors if c in a]
>>> print ci
[]
```

This doesn't require any exception handling to be added anywhere else.